rpc/handler.go

```diff
@@ -19,13 +19,14 @@ package rpc
 import (
 	"context"
 	"encoding/json"
+	"log"
```

Comment:
Was this an intentional or accidental change? We don't need the `log` package, we use our own